src/main/java/com/stickynotes/createsticky/PostController.java

```diff
@@ -0,0 +1,35 @@
+package com.stickynotes.createsticky;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import com.stickynotes.dto.StickyNotesDto;
+import com.stickynotes.pojos.StickyNotesPojo;
+import com.stickynotes.services.CreateStickyNotesService;
+
+@Controller
+public class PostController {
+
+	@Autowired
+	CreateStickyNotesService createStickyNotesService;
+	
+	public ResponseEntity<String> health() {
```

Comment:
missing request mapping

Reply:
I have removed the code as it is unnecessary